### PR TITLE
review: project configuration, documentation, and complexity

### PR DESCRIPTION
**Pull request by ldez**

### CODE REVIEW

**Comment by sivchari** [comment]
thx for PR !

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # sleuth
 
+[![Build Status](https://github.com/ldez/tagliatelle/workflows/Main/badge.svg?branch=master)](https://github.com/ldez/tagliatelle/actions)
```
Comment (butuzov) [inline README.md:3]: @ldez ?

Comment (sivchari) [reply README.md:3]: eh... sorry, I overlooked. I fixed this PR on #13 .

Comment (ldez) [reply README.md:3]: oops

**Pull request merged**